src/less/colors.less

```diff
@@ -58,6 +58,13 @@
 @accent_purple_tint_1: #9244fa;
 @accent_purple_tint_2: #a460ff;
 
+// Label colors:
+@label_green: #b9ecda;
+@label_pink: #ffcbe6;
+@label_purple: #dbc2fe;
+@label_red: #febdc4;
+@label_yellow: #fbe7bc;
```

Comment:
not sure if there's a better naming scheme here (does it make more sense to add it as a tint or something instead?), Jemma seems okay with it though

Reply:
If these are referring to just purely the label colors then I think this naming scheme is fine. If we get different shades of each color then we can consider adding those as tints.